webapp/src/main/java/org/opentestsystem/rdw/admin/model/CsvSubject.java

```diff
@@ -0,0 +1,19 @@
+package org.opentestsystem.rdw.admin.model;
+
+/**
+ * This class represents the Group subjects allowed during CSV Group uploads.
+ */
+public enum CsvSubject {
```

Comment:
Oh. Yeah. 

So, if the subject is null for all records for a group, are we assuming it is like All was specified somewhere? Or is it an error?